Packages/QRScanner/Sources/QRScannerError.swift

```diff
@@ -0,0 +1,23 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+
+enum QRScannerError: Error {
+    case notSupported
+    case permissionsNotGranted
+    case decoding
+    case unexpected(Error)
+
+    func localizedDescription(using provider: QRScannerTextProviding) -> String {
```

Comment:
no need to specify `localizedDescription` here. 

On the app level you can just conform to `extension QRScannerError: LocalizedError` that will have access to localized files